codeforboston/maple
# Adjust desktop add edit testimony

# Summary
Closes #1305 

Adjusted add/edit testimony pages for the desktop according to the new design on Figma https://www.figma.com/file/NPlnWH6AiI2IQMebd8ardO/Mobile-Screens?type=design&node-id=558%3A8200&mode=design&t=k0B4pQcwXIj2PFa8-1 

# Checklist


# Screenshots
![tempsnip](https://github.com/codeforboston/maple/assets/39149005/20222032-c266-4246-861e-41c349ca52ff)
![tempsnip2](https://github.com/codeforboston/maple/assets/39149005/6623fbbf-4708-4a7d-81ec-31c6d3df7f88)
![tempsnip3](https://github.com/codeforboston/maple/assets/39149005/a278f336-6c84-48fd-a1b6-ac8e19d50123)



# Steps to test/reproduce

1. Go any bill.
2. Click "Create Testimony" on the right side of the page.
3. Follow the instructions to complete your testimony to see the changes on those pages.
